src/g_game.c

```diff
@@ -427,14 +427,14 @@ static int CalcControllerSideStrafe(int speed)
   return BETWEEN(-sidemove[speed], sidemove[speed], side);
 }
 
-static double CalcControllerAngle(int speed)
+static double CalcControllerAngle(void)
 {
-  return (angleturn[speed] * axes[AXIS_TURN] * direction[joy_invert_turn]);
+  return (angleturn[1] * axes[AXIS_TURN] * direction[joy_invert_turn]);
 }
 
-static double CalcControllerPitch(int speed)
+static double CalcControllerPitch(void)
 {
-  const double pitch = angleturn[speed] * axes[AXIS_LOOK];
+  const double pitch = angleturn[1] * axes[AXIS_LOOK];
```

Comment:
So this is always fast turning speed?

Reply:
Yes, same as older versions of Woof: https://github.com/fabiangreffrath/woof/blob/f75ebb283f8318f8ed54f1448d1808fce01fb18c/src/g_game.c#L443